### PR TITLE
feat: add coinbase flag to identify mined transactions

### PR DESCRIPTION
**Pull request by alex1307**
Description

Add new field [is_coinbase] to wallet transactions to identify the mined transactions/event.
fixes: #287 

Motivation and Context
We should identify the minning transactions.

How Has This Been Tested?
Shold be tested along with the front-end.

### CODE REVIEW

File: applications/tari_console_wallet/src/grpc/mod.rs [hunk 1]
```diff
@@ -29,7 +29,8 @@ pub fn convert_to_transaction_event(event: String, source: TransactionWrapper) -
             status: completed.status.to_string(),
             direction: completed.direction.to_string(),
             amount: completed.amount.as_u64(),
-            message: completed.message,
+            message: completed.clone().message,
```
Comment (CjS77) [inline applications/tari_console_wallet/src/grpc/mod.rs]: You shouldn't need to clone the whole of `completed` here

File: applications/tari_console_wallet/src/grpc/mod.rs [hunk 2]
```diff
@@ -29,7 +29,8 @@ pub fn convert_to_transaction_event(event: String, source: TransactionWrapper) -
...
+            is_coinbase: (&completed.clone()).is_coinbase(),
```
Comment (CjS77) [inline applications/tari_console_wallet/src/grpc/mod.rs]: Definitely don't need to clone ehre

Comment (alex1307) [reply applications/tari_console_wallet/src/grpc/mod.rs]: Fixed

**Review by CjS77 — APPROVED** [review]
LGTM

**Pull request merged**